README.md

```diff
@@ -141,52 +141,52 @@ The `findAllByMintList` method accepts an array of mint addresses and returns an
 
 ```kotlin
 metaplex.nft.findAllByMintList(listOf(mintPublicKey, mintPublicKey)){ result ->
-	result.onSuccess { nfts ->
-	   val nftList = nfts.filterNotNull() // useful to remove null
-	   ...
-	}
+    result.onSuccess { nfts ->
+       val nftList = nfts.filterNotNull() // useful to remove null
+       ...
+    }
 }
 ```
 
-NFTs retrieved via `findAllByMintList` will not have their JSON metadata loaded because this would require one request per NFT and could be inefficient if you provide a long list of mint addresses. Additionally, you might want to fetch these on-demand, as the NFTs are being displayed on your web app for instance.
+NFTs retrieved via `findAllByMintList` will not have their JSON metadata loaded because this would require one request per NFT. This could be inefficient if you provide a long list of mint addresses. Additionally, you might want to fetch this on-demand, as the NFTs are displayed on your web app, for instance.
```

Comment:
should keep it as "Additionally, you might want to fetch **these** on demand..." since we are talking about a plural subject in this case (list of NFTs)